opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchQueryTest.java

```diff
@@ -73,113 +64,31 @@ public void build_succeeds_with_two_arguments() {
-        namedArgument("query", "query_value"),
-        namedArgument("fuzzy_rewrite", "constant_score")
-    );
-    Assertions.assertNotNull(matchQuery.build(new MatchExpression(arguments)));
-  }
-
-  @Test
-  public void test_lenient_parameter() {
-    List<Expression> arguments = List.of(
-        namedArgument("field", "field_value"),
-        namedArgument("query", "query_value"),
-        namedArgument("lenient", "false")
-    );
-    Assertions.assertNotNull(matchQuery.build(new MatchExpression(arguments)));
-  }
-
-  @Test
-  public void test_operator_parameter() {
-    List<Expression> arguments = List.of(
-        namedArgument("field", "field_value"),
-        namedArgument("query", "query_value"),
-        namedArgument("operator", "OR")
-    );
-    Assertions.assertNotNull(matchQuery.build(new MatchExpression(arguments)));
-  }
-
-  @Test
-  public void test_minimum_should_match_parameter() {
-    List<Expression> arguments = List.of(
-        namedArgument("field", "field_value"),
-        namedArgument("query", "query_value"),
-        namedArgument("minimum_should_match", "3")
-    );
-    Assertions.assertNotNull(matchQuery.build(new MatchExpression(arguments)));
-  }
-
-  @Test
-  public void test_zero_terms_query_parameter() {
-    List<Expression> arguments = List.of(
-        namedArgument("field", "field_value"),
-        namedArgument("query", "query_value"),
-        namedArgument("zero_terms_query", "NONE")
-    );
-    Assertions.assertNotNull(matchQuery.build(new MatchExpression(arguments)));
-  }
-
-  @Test
-  public void test_boost_parameter() {
-    List<Expression> arguments = List.of(
-        namedArgument("field", "field_value"),
-        namedArgument("query", "query_value"),
-        namedArgument("boost", "1")
-    );
-    Assertions.assertNotNull(matchQuery.build(new MatchExpression(arguments)));
+  public void test_valid_parameters() {
+    final int validArgIndex = 2;
+    final List<Expression> validArgs = List.of(
+        namedArgument("analyzer", "standard"),
+        namedArgument("auto_generate_synonyms_phrase_query", "true"),
+        namedArgument("fuzziness", "AUTO"),
+        namedArgument("max_expansions", "50"),
+        namedArgument("prefix_length", "0"),
+        namedArgument("fuzzy_transpositions", "true"),
+        namedArgument("fuzzy_rewrite", "constant_score"),
+        namedArgument("lenient", "false"),
+        namedArgument("operator", "OR"),
+        namedArgument("minimum_should_match", "3"),
+        namedArgument("zero_terms_query", "NONE"),
+        namedArgument("boost", "1"));
+
+    List<Expression> arguments = new ArrayList<Expression>();
+    arguments.add(namedArgument("field", "field_value"));
+    arguments.add(namedArgument("query", "query_value"));
+    arguments.add(namedArgument("", ""));
+
+    for (Expression arg : validArgs) {
+      arguments.set(validArgIndex, arg);
+      Assertions.assertNotNull(matchQuery.build(new MatchExpression(arguments)));
```

Comment:
Let's rework this to use parametrized test.

It will keep execution of tests independent and makes it more obvious what's tested.